modules/account-lib/resources/trx/protobuf/tron.d.ts

```diff
@@ -1,11024 +0,0 @@
-import * as $protobuf from "protobufjs";
```

Comment:
removing these is problematic - i'm not so sure we didn't modify these in previous commits

Reply:
i think we can disregard this